src/NodeVisitor/UsedClassNodeVisitor.php

```diff
@@ -33,10 +33,6 @@ public function beforeTraverse(array $nodes): array
 
     public function enterNode(Node $node): Node|null|int
     {
-        if ($node instanceof FuncCall) {
```

Comment:
Should be changed to $node instanceof Class_ || $node instanceof Function_ || $node instanceof Closure

Reply:
hmm, it seems scan all stmts, It seems need test to cover it tho